README.md

```diff
@@ -1,6 +1,6 @@
 # Exercism GDScript Track
 
-[![configlet](https://github.com/exercism/gdscript/workflows/configlet/badge.svg)](https://github.com/exercism/gdscript/actions?query=workflow%3Aconfiglet) [![tests](https://github.com/exercism/gdscript/workflows/test/badge.svg)](https://github.com/exercism/gdscript/actions?query=workflow%3Atest)
+[![configlet](https://github[![Configlet](https://github.com/exercism/gdscript/actions/workflows/configlet.yml/badge.svg)](https://github.com/exercism/gdscript/actions/workflows/configlet.yml) [![Tests](https://github.com/exercism/gdscript/actions/workflows/test.yml/badge.svg)](https://github.com/exercism/gdscript/actions/workflows/test.yml)
```

Comment:
I think there are some Markdown syntax issues here, the README in your branch doesn't render properly ... Could you please take a look @BNAndras ?